demo/README.md

```diff
@@ -0,0 +1,69 @@
+# ModelDB Demo Site
```

Comment:
Please add a comment here that these instructions are for creating the demo setup as on modeldb.csail.mit.edu. To set up a ModelDB installation on their local machine, point to the root README with those instructions.